demos/common/python/models/deblurring.py

```diff
@@ -20,15 +20,16 @@
 class Deblurring(Model):
     def __init__(self, ie, model_path, input_image_shape):
         super().__init__(ie, model_path)
-        self.calculate_new_shape(input_image_shape)
+        self.block_size = 32
+        self.input_shape = input_image_shape
+        self.calculate_new_shape()
         self.input_blob_name = self.prepare_inputs()
         self.output_blob_name = self.prepare_outputs()
 
-    def calculate_new_shape(self, shape):
-        h, w, _ = shape
-        block_size = 32
-        new_height = math.ceil(h / block_size) * block_size
-        new_width = math.ceil(w / block_size) * block_size
+    def calculate_new_shape(self):
```

Comment:
While you are here, could you rename this method: it not only computes a shape, it actually reshapes and it's what it's called for. The name `reshape` would go for me.